app/models/food_item.py

```diff
@@ -0,0 +1,16 @@
+'''
+FoodItem stores all food item related data
+such as station, name, calories, categories, etc.
+'''
+class FoodItem:
+    def __init__(self, station: str, name: str, calories: int, categories: list):
+        self.station = station
```

Comment:
Should the `FoodItem` be a subclass of pydantic `BaseModel` so that we can add them to the database later? Or that will be handled by the App backend only?

Reply:
I think the app backend will be dealing with this mostly. I can only think of one scenario where we might want this in the dining backend: we move reviews to the dining-backend, which I think we may want to incorporate. In any case, I think the pydantic BaseModel should implement a separate model? The request doesn't care about calories or categories, but rather to identify the food item via name or some other `uuid`. This class is more so for structure.

Wdyt?

Reply:
Yes, I think we want to save the food item to the database, mainly for the review function. We can have a separate model for that.